@stellar/anchor-tests/src/tests/sep38/prices.ts

```diff
@@ -121,7 +121,11 @@ export const hasValidSchema: Test = {
     }
     for (const asset of pricesResponse.buy_assets) {
       const parts = asset.asset.split(":");
-      if (parts.length !== 2 || parts[0] !== "iso4217") {
+      
+      const notValidFiat = parts.length === 2 && parts[0] !== "iso4217";
+      const notValidStellar = parts.length === 3 && parts[0] !== "stellar";
+      const notValidAtAll = parts.length < 2 || parts.length > 3;
+      if (notValidFiat || notValidStellar || notValidAtAll) {
```

Comment:
I made this change because since https://github.com/stellar/stellar-protocol/pull/1149, SEP-31 accepts delivering on-chain access as well